Sources/Application/SceneDelegate.swift

```diff
@@ -20,6 +21,7 @@ class SceneDelegate: UIResponder,
             fatalError("Window failed to initialise in SceneDelegate")
         }
         windowManager = WindowManager(windowScene: windowScene)
+        UIBarButtonItem.appearance(whenContainedInInstancesOf: [UINavigationBar.self]).tintColor = .gdsGreen
```

Comment:
Does this mean we can remove `root.tabBar.tintColor = .gdsGreen` from the MainCoordinator? 👀
Wondering if we can do something similar for `root.tabBar.backgroundColor = .systemBackground` in there as well 🤔

Reply:
It seems to have worked - I've moved them into the SceneDelegate and it seems okay on my end but probably worth checking just to be sure

Reply:
Ahh okay sorry I'm getting it twisted, 26 is for when we've got buttons in navigation bars. Good shout in adding 24 and 25, just moved them to their own private function to simplify this method a bit 👍